e2e-tests/tests/expo.test.ts

```diff
@@ -143,4 +153,4 @@ module.exports = config;`,
     const bundled = await checkIfExpoBundles(projectDir, 'web');
     expect(bundled).toBe(true);
   });
-});
+});
```

Comment:
```suggestion
});

```

I'm surprised `prettier` didn't catch this,